source/Halibut.Tests/TestServices/IReadDataSteamService.cs

```diff
@@ -3,5 +3,7 @@ namespace Halibut.Tests.TestServices
     public interface IReadDataSteamService
     {
         long SendData(DataStream dataStream);
+
+        long SendDataMany(DataStream[] dataStreams);
```

Comment:
suggestion: Is there a reason this needs to be a separate additional API? I suspect we could achieve the same thing and maintain backwards compatibility using `params`

```suggestion
        long SendData(params DataStream[] dataStream);
```

Reply:
Done, thank you!